satpy/tests/reader_tests/test_seviri_l1b_icare.py

```diff
@@ -133,7 +133,7 @@ def test_load_dataset_vis(self):
         datasets = r.load(["VIS008"])
         assert len(datasets) == 1
         for v in datasets.values():
-            dt = datetime(2004, 12, 29, 12, 27, 44)
+            dt = dt.datetime(2004, 12, 29, 12, 27, 44)
```

Comment:
This overrides the module with a timestamp

Reply:
That only fixed the one in the snippet, I'll have a look at what each of the `dt` -> `date` actually mean and update accordinly.

Reply:
And the somewhat better naming in https://github.com/pytroll/satpy/pull/2787/commits/37301177ea7bded84500ca596c82e621a1b1ef4d